libraries/common/src/main/java/androidx/media3/common/ColorInfo.java

```diff
@@ -288,10 +288,27 @@ public Builder buildUpon() {
   /**
    * Returns whether this instance is valid.
    *
-   * <p>This instance is valid if no color members are {@link Format#NO_VALUE}, while bit depths may
-   * be unset.
+   * <p>This instance is valid if at least one between bitdepths and color info are valid.
    */
   public boolean isValid() {
+    return isBitdepthValid() || isColorValid();
+  }
+
+  /**
+   * Returns whether this instance has valid bitdepths.
+   *
+   * <p>This instance has valid bitdepths if none of them is {@link Format#NO_VALUE}.
+   */
+  public boolean isBitdepthValid() {
+    return lumaBitdepth != Format.NO_VALUE && chromaBitdepth != Format.NO_VALUE;
+  }
+
+  /**
+   * Returns whether this instance is color valid.
+   *
+   * <p>This instance is valid if no color members are {@link Format#NO_VALUE}.
+   */
+  public boolean isColorValid() {
```

Comment:
Please change this method to be called isDataspaceValid(). Name comes from the Android context combining this information: https://developer.android.com/reference/android/hardware/DataSpace.

Reply:
Done